packages/blocks/table/src/components/EditableCell.tsx

```diff
@@ -86,7 +86,7 @@ export const EditableCell: VoidFunctionComponent<EditableCellProps> = ({
   return (
     <textarea
       ref={textAreaRef}
-      className={tw`block resize-none w-full bg-transparent rounded border-1 border-transparent hover:border-blue-500 py-1 px-2 -mx-2 `}
+      className={tw`block resize-none w-full bg-transparent rounded border-1 border-transparent hover:border-blue-500 py-1 px-2 -mx-2 w-max max-w-[150px] md:max-w-[300px]`}
```

Comment:
Since we're using textareas for editable cells, we can't have it auto-wrap the content present inside it, ie, width: max-content won't work. Hence, I'm relying on using max-widths (which textarea does listen to) as a way to have sensible default widths for both desktop and mobile.